memcached/README.md

```diff
@@ -0,0 +1,45 @@
+# YCSB Memcached binding
```

Comment:
Are all the bindings not under the same license as YCSB itself (Apache 2.0)? I assume so, and I don't see any other license files in other directories (though I only sampled a few, but all the Java files I saw had Apache 2.0 header).

Is this really necessary? Even just saying that "this binding is licensed under Apache 2.0" may inadvertently imply that repo has a set of licenses, up to one for each binding, which is presumably not the message it should be sending.